### DISCUSSION TITLE
thanks~

### DISCUSSION THREAD

**Discussion by vilmibm** [charmbracelet/huh#401] [General]
This library is great and fulfills a lot of what I wanted to do with prompts in `gh`. I'm especially thrilled about the screen reader mode! That was a goal I never got to before quitting GitHub.

Thanks for the great work! I'm using `huh` for https://tilde.town stuff.

**Comment by meowgorithm** [comment]
Thanks for the kind words, @vilmibm. Funny enough, we actually developed `huh` for `gh`, which is why you're seeing support for screen readers (which I'm also quite thrilled about!).

Let us know if you have any feedback on the package, and don't be a stranger.
